src/Elastic.Apm/Model/Transaction.cs

```diff
@@ -436,6 +451,25 @@ private Activity StartActivity()
 			return activity;
 		}
 
+		internal void UpdateDroppedSpanStats(string destinationServiceResource, Outcome outcome, double duration)
```

Comment:
nitpick: the amount of work that has to be done when a new dropped span stats dictionary is created vs. one that already exists, is different e.g. a new dropped span stats dictionary doesn't need to be checked if count >= 128 or if it contains a specific value, so could branch based on whether dropped span stats dictionary is null or not. It's a minor optimization for sure, but would still be just as readable in my opinion.